doc/manual/manual.bib

```diff
@@ -11526,11 +11526,29 @@ @software{aspect-doi-v2.2.0
   url          = {https://doi.org/10.5281/zenodo.3924604}
 }
 
+
+@software{aspect-doi-v2.3.0,
+  author       = {Wolfgang Bangerth and
+                  Juliane Dannberg and
+                  Menno Fraters and
+                  Rene Gassmoeller and
+                  Anne Glerum and
+                  Timo Heister and
+                  John Naliboff},
+  title        = {ASPECT v2.3.0},
+  month        = jul,
+  year         = 2021,
+  publisher    = {Zenodo},
+  version      = {v2.3.0},
+  doi          = {10.5281/zenodo.5131909},
+  url          = {https://doi.org/10.5281/zenodo.5131909}
+}
+
 @misc{aspectmanual,
   title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}},
-  author        = {W. Bangerth and J. Dannberg and R. Gassm{\"o}ller and T. Heister and others},
-  year          = {2020},
-  month         = {June},
+  author        = {Bangerth, Wolfgang and Dannberg, Juliane and Fraters, Menno and Gassmoeller, Rene and Glerum, Anne and Heister, Timo and Naliboff, John},
```

Comment:
I assume that you have cleared the change in spelling for @gassmoeller 's last name with him.

Reply:
That was not intentional. I did not write this manually, but copied from https://figshare.com/articles/journal_contribution/ASPECT_Advanced_Solver_for_Problems_in_Earth_s_ConvecTion_User_Manual/4865333

@gassmoeller will need to decide how he wants to be named. Zenodo (see above) also uses oe...